packages/cli/src/commands/setup/mailer/mailerHandler.js

```diff
@@ -0,0 +1,119 @@
+import fs from 'fs'
+import path from 'path'
+
+import { Listr } from 'listr2'
+
+import { addApiPackages } from '@redwoodjs/cli-helpers'
+import { errorTelemetry } from '@redwoodjs/telemetry'
+
+import { getPaths, transformTSToJS, writeFile } from '../../../lib'
+import c from '../../../lib/colors'
+import { isTypeScriptProject } from '../../../lib/project'
+
+export const handler = async ({ force, skipExamples }) => {
+  const projectIsTypescript = isTypeScriptProject()
+  const redwoodVersion =
```

Comment:
Seems like many setup needs, getting the RW version feels like it could be a utility function rather than repeated the logic here. I know I had to have it for realtime setup and I think I checked a different package inside the setup/internal package rather than the project package -- which seems more sensible and can be consistent across all setups.